.gitignore

```diff
@@ -90,3 +90,4 @@ NuGetExeBuild/
 NuGetRefBuild/
 Packages/
 NuGetTaskBuild/
+build/
```

Comment:
I personally normally have a `BuildScripts` folder and a `BuildArtifacts` folder.  So I would prefer to see something like that.  Just my 2 cents...